MarkupGatewayPlugin.inc.php

```diff
@@ -257,25 +257,98 @@ function fetch($args, $request) {
 	 */
 	protected function _buildMetadata($journal, $submission)
 	{
-		$locale = AppLocale::getLocale();
-
+		$locale = ($submission->getLanguage() != '') ? $submission->getLanguage() : $journal->getPrimaryLocale();
+		
+		$publishedArticleDao = DAORegistry::getDAO('PublishedArticleDAO');
+		$issueDao = DAORegistry::getDAO('IssueDAO');		
+		$sectionDao = DAORegistry::getDAO('SectionDAO');
+		$userGroupDao = DAORegistry::getDAO('UserGroupDAO');
+		
+		/* Authors */
 		$authors = array_map(function($author)
 		{
 			return array (
-					'name' => $author->getFullName(),
+					'firstName' => $author->getFirstName(),
+					'lastName' => $author->getLastName(),
 					'email' => $author->getEmail(),
+					'orcid' => $author->getOrcid(),
+					'affiliation' => $author->getLocalizedAffiliation(),
+					'country' => $author->getCountry(),
+					'bio' => $author->getLocalizedBiography(),
+					'contribType' => $userGroupDao->getById($author->getUserGroupId()),
```

Comment:
Sorry, I have been busy since I returned to work on my phd so I am doing OJS related work on my spare time. I should be able to check this next week. I have this and the fundref plugin in my todo list.